src/servicesHost.ts

```diff
@@ -69,7 +69,9 @@ export function makeServicesHost(
     fileExists,
     readFile: readFileWithFallback,
     realpath: compiler.sys.realpath,
-    directoryExists: compiler.sys.directoryExists
+    directoryExists: compiler.sys.directoryExists,
+    getCurrentDirectory: compiler.sys.getCurrentDirectory,
+    getDirectories: compiler.sys.getDirectories
   };
```

Comment:
So if I understand this correctly, the fix is the implementation of these: 

```
    directoryExists: compiler.sys.directoryExists,
    getCurrentDirectory: compiler.sys.getCurrentDirectory,
    getDirectories: compiler.sys.getDirectories
```
?

That's interesting.  I wish it was more "fail fast" here if it actually needed these...

Reply:
Yep, specifically `getCurrentDirectory`. I agree. I'll suggest something like this to the team.